test/connectors/CompanyHouseConnectorSpec.scala

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import base.SpecBase
+import models.{Submission, SubmissionResponse}
+import org.scalatest.concurrent.ScalaFutures
+import play.api.libs.json.{JsValue, Json}
+import uk.gov.hmrc.http.{HeaderCarrier, HttpResponse, UpstreamErrorResponse}
+import uk.gov.hmrc.play.bootstrap.http.HttpClient
+import utils.MockUserAnswers
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+import org.mockito.Mockito._
+import org.mockito.ArgumentMatchers.{any, eq => eqTo}
+
+class CompanyHouseConnectorSpec extends SpecBase with ScalaFutures {
+
+  "submission" must {
+
+    "return an true when the HTTP call to CoHo is a match" in {
```

Comment:
Getting a bit nitpicky now but this should be "return a true" or just "return true"